script/install-svn.sh

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
 
 SVN_BASE=subversion-1.9.7
+MIRROR_PRIMARY=http://www-us.apache.org/dist
+MIRROR_SECONDARY=http://www-eu.apache.org/dist
```

Comment:
@mishaschwartz You're missing the final /subversion dir

Reply:
@adisandro I think I had it lower down initially but I must have removed it. I'll fix it now.